scheme/psa-iot/test/ComidPsaRefValMultDigest.json

```diff
@@ -21,7 +21,7 @@
         "environment": {
           "class": {
             "id": {
-              "type": "PSA_IOT.impl-id",
+              "type": "psa.impl-id",
```

Comment:
is there a specific reason for this change?

This kind of breaks the nice alignment of the Store `key`-`value` symmetry?

Please reconsider!

Reply:
This change is required by `corim` (I think I mentioned this in our discussions earlier?). For whatever reason, `corim` templates explect `psa`. This is hard-coded: 

https://github.com/veraison/corim/blob/main/comid/classid.go#L168

I agree that we should align how we represent things in the stores vs the templeas (and better yet, we should get rid of these entirely unnecessary prefixes). However that would require making further changes to `corim` that orthogonal to what is being implemented here.